### PR TITLE
Allow single quotes [',"] in comments in TextEdit

### LINKED ISSUE #36638: Auto-completion in comments not entirely useful for apostrophes.
**Godot version:** 3.2.0 stable

**OS/device including version:** N/A

**Issue description:** Auto-completion has its issues when you're writing comments. For example `# Can't and won't do this.` would duplicate the `'` character (i.e. treat it as a single quote). This is a minor issue, but makes contractions (i.e. "it's", "can't" etc) slightly more difficult than it needs to be. Maybe look at the way other projects such as Inform 7 (for example) handle this?

**Steps to reproduce:** Just press `'`.

### PR DESCRIPTION
**Pull request by dreamsComeTrue**
....instead of duplicating them on single keystroke.

Fixes: #36638

### CODE REVIEW

**Review by Paulb23 — APPROVED** [review]
LGTM.

File: scene/gui/text_edit.cpp [hunk 1]
```diff
@@ -1880,6 +1884,14 @@ void TextEdit::_consume_pair_symbol(CharType ch) {
 		}
 	}
 
+	//	Do not need to duplicate quotes while in comments
```
Comment (akien-mga) [inline scene/gui/text_edit.cpp]: Should be a space after `//`, not a tab. Same in line 1895 from previous PR.

Comment (dreamsComeTrue) [reply scene/gui/text_edit.cpp]: Fixed, leason learned :)

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.2.

**Pull request merged**